src/canPDC.cpp

```diff
@@ -0,0 +1,21 @@
+#include "canPDC.h"
+
+CANPDC::CANPDC(CAN_TypeDef* canPort, CAN_PINS pins, int frequency) : CANManager(canPort, pins, frequency) {};
+
+void CANPDC::readHandler(CAN_message_t msg) {
+    
+}
+
+void CANPDC::sendPDCData() {
+    readAnalog();
+    this->sendMessage(0x200, (void*)acc_out, sizeof(float));
```

Comment:
I think it has to be `(void*)&acc_out`. The method requires a pointer for the 2nd argument.